lib/cli/commands/subscribealerts.ts

```diff
@@ -0,0 +1,72 @@
+import { ServiceError, status } from 'grpc';
+import { Arguments, Argv } from 'yargs';
+import { XudClient } from '../../proto/xudrpc_grpc_pb';
+import * as xudrpc from '../../proto/xudrpc_pb';
+import { setTimeoutPromise } from '../../utils/utils';
+import { loadXudClient } from '../command';
+import { AlertType } from '../../constants/enums';
+
+export const command = 'subscribealerts [--pretty]';
+
+export const describe = 'subscribe alerts such as low balance';
+
+export const builder = (argv: Argv) => argv
+    .option('pretty', {
+      type: 'boolean',
+    })
+    .example('$0 subscribealerts', 'prints alert payload in a JSON structure')
+    .example('$0 subscribealerts --pretty', 'prints alert message only');
+
+export const handler = async (argv: Arguments) => {
+  await ensureConnection(argv, true);
+};
+
+let client: XudClient;
+
+const ensureConnection = async (argv: Arguments, printError?: boolean) => {
```

Comment:
This is copied from `streamorders.ts` right? It'd be nice in that case to put this in the command utils file and have it accept a callback (`subscribeAlerts` in this case) so that we're not duplicating code.